code/procs/mob_procs.dm

```diff
@@ -1094,8 +1094,18 @@
 	var/rendered = "<span class='game say'>[my_name] <span class='message'>[message_a]</span></span>"
 
 	var/rendered_outside = null
-	if (olocs.len)
-		var/atom/movable/OL = olocs[olocs.len]
+	if (length(olocs))
+		var/atom/movable/OL = olocs[length(olocs)]
+
+		var/obj/head_on_spike/H = locate() in olocs
```

Comment:
variable names pls

Reply:
changed the variable names to obj/O and atom/movable/AM, and atom/A

Reply:
that's arguably worse, please just give them readable names instead of single letter names with comments explaining what they do